### PR TITLE
Remove sensitive data from component logs

### PR DESCRIPTION
**Pull request by denyshld**

### CODE REVIEW

File: src/main/java/io/elastic/jdbc/actions/CreateOrUpdateRecord.java [hunk 1]
```diff
@@ -151,7 +151,7 @@ private boolean recordExists(String tableName, String idColumn, String idValue)
         String query = "SELECT COUNT(*) FROM " + tableName + " WHERE " + idColumn + " = ?";
         PreparedStatement statement = connection.prepareStatement(query);
         setStatementParam(statement, 1, idColumn, idValue);
-        logger.info("{}",statement);
+        logger.debug("{}",statement);
```
Comment (shulkaolka) [inline src/main/java/io/elastic/jdbc/actions/CreateOrUpdateRecord.java:154]:
```suggestion
        logger.trace("{}",statement);
```

File: src/main/java/io/elastic/jdbc/actions/DeleteRowByPrimaryKey.java [hunk 2]
```diff
@@ -75,23 +75,23 @@ public void execute(ExecutionParameters parameters) {
         LOGGER.info("Executing delete row by primary key action");
         boolean isOracle = dbEngine.equals(Engines.ORACLE.name().toLowerCase());
         Utils.columnTypes = Utils.getColumnTypes(connection, isOracle, tableName);
-        LOGGER.info("Detected column types: " + Utils.columnTypes);
+        LOGGER.debug("Detected column types: " + Utils.columnTypes);
         try {
           QueryFactory queryFactory = new QueryFactory();
           Query query = queryFactory.getQuery(dbEngine);
-          LOGGER.info("Lookup parameters: {} = {}", primaryKey.toString(), primaryValue.toString());
+          LOGGER.debug("Lookup parameters: {} = {}", primaryKey.toString(), primaryValue.toString());
```
Comment (shulkaolka) [inline src/main/java/io/elastic/jdbc/actions/DeleteRowByPrimaryKey.java:82]:
```suggestion
          LOGGER.trace("Lookup parameters: {} = {}", primaryKey.toString(), primaryValue.toString());
```

File: src/main/java/io/elastic/jdbc/actions/DeleteRowByPrimaryKey.java [hunk 3]
```diff
@@ -75,23 +75,23 @@ public void execute(ExecutionParameters parameters) {
...
           query.from(tableName).lookup(primaryKey.toString(), primaryValue.toString());
           checkConfig(configuration);
           JsonObject row = query.executeLookup(connection, body);
 
           for (Map.Entry<String, JsonValue> entry : configuration.entrySet()) {
-            LOGGER.info("Key = " + entry.getKey() + " Value = " + entry.getValue());
+            LOGGER.debug("Key = " + entry.getKey() + " Value = " + entry.getValue());
```
Comment (shulkaolka) [inline src/main/java/io/elastic/jdbc/actions/DeleteRowByPrimaryKey.java:88]:
```suggestion
            LOGGER.trace("Key = " + entry.getKey() + " Value = " + entry.getValue());
```

File: src/main/java/io/elastic/jdbc/actions/DeleteRowByPrimaryKey.java [hunk 4]
```diff
@@ -124,11 +124,11 @@ public void execute(ExecutionParameters parameters) {
               return;
             }
           }
-          LOGGER.error("Failed to make request", e.toString());
+          LOGGER.error("Failed to make request {}", e.toString());
```
Comment (shulkaolka) [inline src/main/java/io/elastic/jdbc/actions/DeleteRowByPrimaryKey.java:127]:
```suggestion
          LOGGER.error("Failed to make the request.");
```

File: src/main/java/io/elastic/jdbc/actions/LookupRowByPrimaryKey.java [hunk 5]
```diff
@@ -67,7 +67,7 @@ public void execute(ExecutionParameters parameters) {
     boolean isOracle = dbEngine.equals(Engines.ORACLE.name().toLowerCase());
 
     for (Map.Entry<String, JsonValue> entry : body.entrySet()) {
-      LOGGER.info("{} = {}", entry.getKey(), entry.getValue());
+      LOGGER.debug("{} = {}", entry.getKey(), entry.getValue());
```
Comment (shulkaolka) [inline src/main/java/io/elastic/jdbc/actions/LookupRowByPrimaryKey.java:70]:
```suggestion
      LOGGER.trace("{} = {}", entry.getKey(), entry.getValue());
```

File: src/main/java/io/elastic/jdbc/actions/LookupRowByPrimaryKey.java [hunk 6]
```diff
@@ -78,7 +78,7 @@ public void execute(ExecutionParameters parameters) {
       try (Connection connection = Utils.getConnection(configuration)) {
         LOGGER.info("Executing lookup row by primary key action");
         Utils.columnTypes = Utils.getColumnTypes(connection, isOracle, tableName);
-        LOGGER.info("Detected column types: " + Utils.columnTypes);
+        LOGGER.trace("Detected column types: " + Utils.columnTypes);
```
Comment (shulkaolka) [inline src/main/java/io/elastic/jdbc/actions/LookupRowByPrimaryKey.java:81]:
```suggestion
        LOGGER.debug("Detected column types: " + Utils.columnTypes);
```

File: src/main/java/io/elastic/jdbc/providers/ColumnNamesForInsertProvider.java [hunk 7]
```diff
@@ -61,10 +61,10 @@ private JsonObject getInputMetaData(JsonObject configuration) {
     try (Connection connection = Utils.getConnection(configuration)) {
       DatabaseMetaData dbMetaData;
       try {
-        LOGGER.info("Getting DatabaseMetaData for table: '{}'...", tableName);
+        LOGGER.debug("Getting DatabaseMetaData for table: '{}'...", tableName);
```
Comment (shulkaolka) [inline src/main/java/io/elastic/jdbc/providers/ColumnNamesForInsertProvider.java:64]:
```suggestion
        LOGGER.trace("Getting DatabaseMetaData for table: '{}'...", tableName);
```

File: src/main/java/io/elastic/jdbc/providers/ColumnNamesForInsertProvider.java [hunk 8]
```diff
@@ -61,10 +61,10 @@ private JsonObject getInputMetaData(JsonObject configuration) {
...
         dbMetaData = connection.getMetaData();
       } catch (SQLException e) {
-        LOGGER.error("Failed while getting DatabaseMetaData. Error: " + e.getMessage());
+        LOGGER.debug("Failed while getting DatabaseMetaData. Error: " + e.getMessage());
```
Comment (shulkaolka) [inline src/main/java/io/elastic/jdbc/providers/ColumnNamesForInsertProvider.java:67]:
```suggestion
        LOGGER.error("Failed while getting DatabaseMetaData ");
```

File: src/main/java/io/elastic/jdbc/providers/ColumnNamesForInsertProvider.java [hunk 9]
```diff
@@ -78,21 +78,21 @@ private JsonObject getInputMetaData(JsonObject configuration) {
         final String catalog = isMySql ? configuration.getString("databaseName") : null;
         ArrayList<String> primaryKeysNames = Utils
             .getPrimaryKeyNames(catalog, schemaNamePattern, tableNamePattern, dbMetaData);
-        LOGGER.info("Found primary key name(s): '{}'", primaryKeysNames);
+        LOGGER.debug("Found primary key name(s): '{}'", primaryKeysNames);
```
Comment (shulkaolka) [inline src/main/java/io/elastic/jdbc/providers/ColumnNamesForInsertProvider.java:81]:
```suggestion
        LOGGER.trace("Found primary key name(s): '{}'", primaryKeysNames);
```

File: src/main/java/io/elastic/jdbc/providers/ColumnNamesForInsertProvider.java [hunk 10]
```diff
@@ -78,21 +78,21 @@ private JsonObject getInputMetaData(JsonObject configuration) {
...
 
         LOGGER.info("Starting processing columns...");
         while (resultSet.next()) {
           final String fieldName = resultSet.getString("COLUMN_NAME");
           final int sqlDataType = resultSet.getInt("DATA_TYPE");
           final String fieldType = Utils.convertType(sqlDataType);
-          LOGGER.info("Found column: name={}, type={}", fieldName, fieldType);
+          LOGGER.debug("Found column: name={}, type={}", fieldName, fieldType);
```
Comment (shulkaolka) [inline src/main/java/io/elastic/jdbc/providers/ColumnNamesForInsertProvider.java:88]:
```suggestion
          LOGGER.trace("Found column: name={}, type={}", fieldName, fieldType);
```

File: src/main/java/io/elastic/jdbc/providers/QueryColumnNamesProvider.java [hunk 11]
```diff
@@ -53,8 +53,8 @@ public JsonObject getColumns(JsonObject configuration) {
     if (matcher.find()) {
       do {
         matcherCheckCharacter.find();
-        LOGGER.info("Var = {}", matcher.group());
-        LOGGER.info("Var matcherCheckCharacter = {}", matcherCheckCharacter.group());
+        LOGGER.debug("Var = {}", matcher.group());
+        LOGGER.debug("Var matcherCheckCharacter = {}", matcherCheckCharacter.group());
```
Comment (shulkaolka) [inline src/main/java/io/elastic/jdbc/providers/QueryColumnNamesProvider.java:57]:
```suggestion
        LOGGER.trace("Var = {}", matcher.group());
        LOGGER.trace("Var matcherCheckCharacter = {}", matcherCheckCharacter.group());
```

File: src/main/java/io/elastic/jdbc/utils/UtilsOld.java [hunk 12]
```diff
@@ -30,7 +30,8 @@ public static Connection getConnection(final JsonObject config) {
         logger.info("DB type {}", engineType);
         engineType.loadDriverClass();
         final String connectionString = engineType.getConnectionString(host, port, databaseName);
-        logger.info("Connecting to {}", connectionString);
+        logger.info("Connecting to {}", host);
```
Comment (shulkaolka) [inline src/main/java/io/elastic/jdbc/utils/UtilsOld.java:33]:
```suggestion
        logger.trace("Connecting to {}", host);
```

**Review by shulkaolka — CHANGES_REQUESTED** [review]

File: src/main/java/io/elastic/jdbc/utils/UtilsOld.java (hunk 12 above)
Comment (denyshld) [reply src/main/java/io/elastic/jdbc/utils/UtilsOld.java:33]: Url is not so valuable info, nut it can be useful for common logs

**Review by shulkaolka — APPROVED** [review]

**Pull request merged**